packages/forms/src/UIForm/fields/CheckBox/CheckBoxes.component.js

```diff
@@ -51,15 +51,15 @@ if (process.env.NODE_ENV !== 'production') {
 		id: PropTypes.string,
 		isValid: PropTypes.bool,
 		errorMessage: PropTypes.string,
-		onChange: PropTypes.func,
+		onChange: PropTypes.func.isRequired,
 		schema: PropTypes.shape({
 			description: PropTypes.string,
 			title: PropTypes.string,
-			titleMap: PropTypes.shape({
+			titleMap: PropTypes.arrayOf(PropTypes.shape({
 				name: PropTypes.string,
 				value: PropTypes.string,
-			}),
-		}),
+			})),
+		}).isRequired,
```

Comment:
schema is required and none of its component is, isn't it more safe then to rely on a default props set to an empty literral object ?